### ISSUE TITLE
Some currency doesn't correct

### ISSUE THREAD

**Issue by khsing** [rinvex/countries#52]
There are 4 countries  returns incorrect Currency.

```
>>> country('us')->getCurrency()
=> [
     "iso_4217_code" => "USN",
     "iso_4217_numeric" => 997,
     "iso_4217_name" => "United States dollar (next day) (funds code)",
     "iso_4217_minor_unit" => 2,
   ]
>>> country('us')->getCurrencies()
=> [
     "USN" => [
       "iso_4217_code" => "USN",
       "iso_4217_numeric" => 997,
       "iso_4217_name" => "United States dollar (next day) (funds code)",
       "iso_4217_minor_unit" => 2,
     ],
     "USS" => [
       "iso_4217_code" => "USS",
       "iso_4217_numeric" => 998,
       "iso_4217_name" => "United States dollar (same day) (funds code)",
       "iso_4217_minor_unit" => 2,
     ],
     "USD" => [
       "iso_4217_code" => "USD",
       "iso_4217_numeric" => 840,
       "iso_4217_name" => "US Dollar",
       "iso_4217_minor_unit" => 2,
     ],
   ]
```
Should return `USD`, but first one  `USN` returned.
```
country('us')->getCurrency()
country('cl')->getCurrency()
country('uy')->getCurrency()
country('bo')->getCurrency()
```
there are 4 countries return incorrect currency.

**Comment by Omranic** [comment]
@khsing Check this please & tell me what do you think http://forexesteem.blogspot.com.eg/2009/03/explanation-of-usdussusn.html

**Comment by khsing** [comment]
@Omranic In general the currency of USA is USD, although USN and USS are more detailed currency code, but not normal. 

I think `getCurrency` should return the Currency which people used to use.

**Comment by Omranic** [comment]
Yes, I agree. We may have to change `getCurrency` behavior and omit or separate the details currencies into different attribute for all countries. I'll keep this issue opened until further review.

**Comment by willemwollebrants** [comment]
I had the same problem. I've fixed it by adding an optional key 'default_currency' to the country data, and a new Country::getDefaultCurrency method. If default_currency is not set, Country::getDefaultCurrency returns the same as Country::getCurrency. (I could have just altered the order of currencies in the json files, but explicitly defining it seems more secure)

[My code's here](https://github.com/rinvex/country/compare/develop...willemwollebrants:defaultCurrency), if it's ok I can send a PR for this

**Comment by markitosgv** [comment]
+1 for USD instead of USN

**Comment by Omranic** [comment]
Thank you for your patience, this is now fixed fundamentaly for all countries in https://github.com/rinvex/countries/commit/1cf372b29bc0c17eeaf08e40a7cd98b729aae296, here's what's been done:

Update country currencies to include only the official currency (fix #52 #55 #72)
- Removed fund, complementary and secondary currencies as it's not relevant to the countries package, and caused confusion before
- This change affected 15 countries: Bolivia, Bhutan, Belarus, Switzerland, Chile, Cuba, Western Sahara, Haiti, British Indian Ocean Territory, Lesotho, Namibia, Panama, El Salvador, United States, Uruguay
